app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -18,49 +18,81 @@ class GetUspsProofingResultsJob < ApplicationJob
 
   discard_on GoodJob::ActiveJobExtensions::Concurrency::ConcurrencyExceededError
 
+  def enrollment_attributes(enrollment, enrollment_completed)
+    {
+      enrollment_code: enrollment.enrollment_code,
+      enrollment_id: enrollment.id,
+      minutes_since_last_status_check: enrollment.minutes_since_last_status_check,
+      minutes_since_last_status_update: enrollment.minutes_since_last_status_update,
+      minutes_to_completion: enrollment_completed ? enrollment.minutes_since_established : nil,
+    }
+  end
+
   def perform(_now)
     return true unless IdentityConfig.store.in_person_proofing_enabled
 
-    proofer = UspsInPersonProofing::Proofer.new
-
+    @enrollment_outcomes = {
+      enrollments_checked: 0,
+      enrollments_errored: 0,
+      enrollments_expired: 0,
+      enrollments_failed: 0,
+      enrollments_in_progress: 0,
+      enrollments_passed: 0,
+    }
     reprocess_delay_minutes = IdentityConfig.store.
       get_usps_proofing_results_job_reprocess_delay_minutes
-    InPersonEnrollment.needs_usps_status_check(
+    enrollments = InPersonEnrollment.needs_usps_status_check(
       ...reprocess_delay_minutes.minutes.ago,
-    ).each do |enrollment|
-      # Record and commit attempt to check enrollment status to database
-      enrollment.update(status_check_attempted_at: Time.zone.now)
+    )
+
+    analytics.idv_in_person_usps_proofing_results_job_started(
+      enrollments_count: enrollments.count,
+      reprocess_delay_minutes: reprocess_delay_minutes,
+    )
+
+    check_enrollments(enrollments)
+
+    analytics.idv_in_person_usps_proofing_results_job_completed(**enrollment_outcomes)
+
+    true
+  end
 
+  private
+
+  attr_accessor :enrollment_outcomes
+
+  DEFAULT_EMAIL_DELAY_IN_HOURS = 1
```

Comment:
I see this just got moved and didn't change, but it seems like a good candidate for a config value we could override

Reply:
Yeah, this was my mistake, I should've just used the config value.

Reply:
There is also an associated config value ([in_person_results_delay_in_hours](https://github.com/18F/identity-idp/blob/eb2f0afe8cb312ed6ad3f2c4799562922d2fc02b/app/jobs/get_usps_proofing_results_job.rb#L249)) and this value is just a fallback in case the config value is unset or negative

Should we just trust that the config value is set and not [handle the condition that it's unset/negative](https://github.com/18F/identity-idp/blob/eb2f0afe8cb312ed6ad3f2c4799562922d2fc02b/app/jobs/get_usps_proofing_results_job.rb#L255)?

Going to move ahead with this PR but can address this in a small follow-up if we want to